virtgpu.c

```diff
@@ -49,6 +52,14 @@ static int virtgpu_init(struct driver *drv)
 		int ret = drmIoctl(drv->fd, DRM_IOCTL_VIRTGPU_GETPARAM, &get_param);
 		if (ret)
 			drv_logi("virtgpu backend not enabling %s\n", params[i].name);
+		if (ret == 0) {
+			if (strcmp(params[i].name, "VIRTGPU_PARAM_QUERY_DEV") == 0) {
+				dev_feature += params[i].value;
+			}
+			if (strcmp(params[i].name, "VIRTGPU_PARAM_RESOURCE_BLOB") == 0) {
+				dev_feature += params[i].value;
+			}
```

Comment:
suggest use bitwise combination for check, for example:
#define VIRTGPU_PARAM_QUERY_DEV_BIT                (1ull << VIRTGPU_PARAM_QUERY_DEV)
			if (strcmp(params[i].name, "VIRTGPU_PARAM_QUERY_DEV") == 0 && params[i].value == 1) {
				dev_feature |= VIRTGPU_PARAM_QUERY_DEV_BIT;
			}